src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -8681,6 +8697,25 @@ public void PathMapParser()
             Assert.Equal((int)ErrorCode.ERR_InvalidPathMap, parsedArgs.Errors[0].Code);
         }
 
+        [WorkItem(7588, "https://github.com/dotnet/roslyn/issues/7588")]
+        [Fact]
+        public void Version()
```

Comment:
The test would be valuable because they would verify that compiler behaves as expected now and will continue doing so in the future when those additional flags are specified. Feel free to throw other kinds of switches into the mix. The two mentioned are especially important because they directly affect the output from the compiler.

---
In reply to: [80532729](https://github.com/dotnet/roslyn/pull/13955#discussion_r80532729) [](ancestors = 80532729)